ibis/backends/pandas/client.py

```diff
@@ -80,11 +80,26 @@
 
 
 _inferable_pandas_dtypes = {
-    'boolean': dt.boolean,
     'string': dt.string,
-    'unicode': dt.string,
     'bytes': dt.string,
-    'empty': dt.string,
+    'floating': dt.float64,
+    'integer': dt.int64,
+    'mixed-integer': dt.binary,
+    'mixed-integer-float': dt.float64,
+    'decimal': dt.binary,
+    'complex': dt.binary,
+    'categorical': dt.binary,
```

Comment:
also can you make sure we have tests that hit each of these if we do not already

Reply:
Just added tests for these in `ibis/backends/pandas/tests/test_datatypes.py`